pkg/localcontroller/util/util.go

```diff
@@ -105,3 +109,23 @@ func AddPrefixPath(prefix string, path string) string {
 func GetUniqueIdentifier(namespace string, name string, kind string) string {
 	return fmt.Sprintf("%s/%s/%s", namespace, kind, name)
 }
+
+// CreateTemporaryDir creates a temporary dir
+func CreateTemporaryDir() (string, error) {
+	var src = rand.NewSource(time.Now().UnixNano())
+	dir := path.Join("/tmp/", strconv.FormatInt(src.Int63(), 10), "/")
+	err := CreateFolder(dir)
+	return dir, err
+}
+
+// ParsingDatasetIndex parses index file of dataset and adds the prefix to abs url of sample
+func ParsingDatasetIndex(samples []string, prefix string) []string {
+	var l []string
+	l = append(l, prefix)
+	for _, v := range samples {
```

Comment:
add some comment about the format of samples here

Reply:
fix it